docs/_posts/murat-gunay/2022-01-14-bert_sequence_classifier_emotion_en.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: model
+title: Emotion Detection Classifier
+author: John Snow Labs
+name: bert_sequence_classifier_emotion
+date: 2022-01-14
+tags: [bert_for_sequence, en, emotion, open_source]
+task: Text Classification
+language: en
+edition: Spark NLP 3.3.4
+spark_version: 3.0
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model was imported from `Hugging Face` and it's been fine-tuned on emotion [dataset](https://huggingface.co/nlp/viewer/?dataset=emotion), leveraging `Bert` embeddings and `BertForSequenceClassification` for text classification purposes.
```

Comment:
I think here the link should be to the model (nateraw/bert-base-uncased-emotion) not the dataset

Reply:
@josejuanmartinez 
I changed the link